pr_agent/settings/configuration.toml

```diff
@@ -26,6 +26,7 @@ inline_code_comments = false
 ask_and_reflect=false
 automatic_review=true
 remove_previous_review_comment=false
+persistent_comment=true
```

Comment:
I think the only potential conflict between these 2 configurations is of they're both true and you run a regular (non-incremental) review.

In this case, the previous comment will be first updated and then deleted. Maybe add a comment warning against this combination of configurations? Or add some assertion in the code?

In essence, each configuration when used separetly, achieves a similar idea of keeping only 1 review comment. The difference is how: edit vs create+delete.  
I think there is still some merit to using `remove_preious_review_comment` for "updating" an existing `/review` comment:
1. It posts the comment on the relevant (reviewed) commit and not the first reviewed commit.
2. It moves the review comment "lower" in the conversation.
3. It triggers a notification (as far as I know editing a comment does not)

Reply:
i will implement your previous advice (publish_persistent_commment)

Maybe for the future try to think if you can use this mechanism also for controlling the incremental review.

it makes more sense to _edit_ existing comments than to add-and-delete.
it also preserves history, and its more traceable